### ISSUE TITLE
Are more than one MQTT Topic supported under a tenant?

### ISSUE THREAD

**Issue by yuyewuye** [sitewhere/sitewhere#848]
Hi，Can I send a class of devices to a MQTT Topic?Instead of sending all the devices under one tenant to a MQTT Topic.
Because i have so many getways in a tenant .

**Comment by derekadams** [comment]
We don't currently make the device type token/id available in the substitution parameters, but I'll use this ticket as a feature request to update that.

**Comment by yuyewuye** [comment]
Can you tell me the time ?I don't want to wait too hard >-<

**Comment by derekadams** [comment]
For SiteWhere 3.0 or 2.x? For 3.0 it's easy since we added a new framework for injecting tokens. The backport to 2.x will take longer and probably will not happen until after the 3.0 beta images are released.

**Comment by yuyewuye** [comment]
I'm really looking forward to the 3.0 release, even if it's late.I suggest that you test the feature carefully before you release it, or at least make sure it is usable. I've been using it since version 1.x ,for 2.X, His reliability and stability are not as good as 1.x .
Technical problem or time problem or team width problem?Finally, what are the new features of 3.0?

**Comment by derekadams** [comment]
Yes, SiteWhere 2.x is a transitional release with the long-term vision being to get to our 3.0 architecture, which is much more integrated with Kubernetes. For a list of some of the features in 3.0, see this post: 

https://github.com/sitewhere/sitewhere/issues/789#issuecomment-556035861 

We will release a beta for 3.0 in order to get feedback and fix bugs to assure that the 3.0 GA release is stable and all features working. As we near the GA release, we will release a complete list of the new features along with improved documentation.

**Comment by yuyewuye** [comment]
Will the bug fix response speed up when version 3.0 is released?And is there a commercial partnership or capital injection with other companies for version 3.0 now?

**Comment by derekadams** [comment]
The team has been focused on developing 3.0 for the last 9 months so bug fixes have been slower than usual. Any issues with 3.0 will be addressed quickly once all the core functionality is considered stable. Regarding commercial partnerships and capital, please reach out to me directly on our Discord server (user handle `derekadams`) and I can clarify those aspects.

**Comment by yuyewuye** [comment]
Is 3.0 open source?

**Comment by derekadams** [comment]
Yes, 3.0 is completely open source under the same license used in 1.x and 2.x. We will have an Enterprise Edition offering in addition to the open source. It is focused on launching/configuring/monitoring SiteWhere instances on various cloud providers such as Google Cloud.
